index.html

```diff
@@ -386,18 +384,6 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
       <dd>
         Synonym for <a>WoT Core Profile</a>.
       </dd>
```

Comment:
agree to remove

Reply:
https://github.com/w3c/wot-profile/pull/116